systemtest/src/test/java/io/strimzi/systemtest/kafka/TieredStorageST.java

```diff
@@ -143,9 +148,9 @@ void testTieredStorageWithAivenPlugin() {
             .build());
 
         final KafkaClients clients = ClientUtils.getInstantPlainClientBuilder(testStorage)
-            .withMessageCount(10000)
+            .withMessageCount(MESSAGE_COUNT)
             .withDelayMs(1)
-            .withMessage(String.join("", Collections.nCopies(5000, "#")))
+            .withMessage(String.join("", Collections.nCopies(300, "#")))
```

Comment:
With 300 bytes * 10000 messages = 3,000,000 bytes. Each segment byte = 1,048,576. So we should roll 2 times with total 3 log segments. That's enough for the tests.